analytics/src/analytics/cli.py

```diff
@@ -122,6 +126,29 @@ def calculate_sprint_burnup(
     )
 
 
+@export_app.command(name="test_connection")
+def test_connection() -> None:
+    """Test function that ensures the DB connection works."""
+    engine = db.get_db()
+    # connection method from sqlalchemy
+    connection = engine.connect()
+
+    # Test INSERT INTO action
+    result = connection.execute(
+        text(
+            "INSERT INTO audit_log (topic,timestamp, end_timestamp, user_id, details)"
+            "VALUES('test','2024-06-11 10:41:15','2024-06-11 10:54:15',87654,'test from command');",
```

Comment:
```suggestion
            "INSERT INTO audit_log (topic,timestamp, end_timestamp, user_id, details) "
            "VALUES('test','2024-06-11 10:41:15','2024-06-11 10:54:15',87654,'test from command');",
```

Multiline strings don't turn newlines into spaces by default, so you need to add a trailing space on each line